### DISCUSSION TITLE
Feature request: sort by preference

### DISCUSSION THREAD

**Discussion by ados8** [docmost/docmost#1622] [Ideas]
I would really like to have spaces sort based on admin preference.
This could be by alphabetical, modified time, manual etc.
For me mainly after alphabetical as at the moment I have to manually move them into that order.

**Comment by sibelius** [comment]
I think sort by alphabetically could help a lot

Where is the code that handle this?

**Comment by sibelius** [comment]
how the position works ?

**Comment by sibelius** [comment]
hey @Philipinho can you help me understand how position works?
